src/reducers/filterReducer.js

```diff
@@ -0,0 +1,12 @@
+import * as filterActions from '../actions/filterActions'
+
+export const filter = (defaultFilter) => (state = defaultFilter, action) => {
+    switch (action.type) {
+        case filterActions.CHANGE_EVENT_FILTER:
+            return Object.assign({ eventTypes: [], ticketId: null }, action.filter)
```

Comment:
Use spread operator (...) to maintain consistency with other reducers.

Reply:
done